specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/ManagedInstanceAdministratorListByInstance.json

```diff
@@ -0,0 +1,27 @@
+{
+    "parameters": {
+        "subscriptionId": "00000000-1111-2222-3333-444444444444",
+        "resourceGroupName": "Default-SQL-SouthEastAsia",
+        "managedInstanceName": "managedInstance",
+        "api-version": "2017-03-01-preview"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "value": [
+                    {
```

Comment:
>{ [](start = 20, length = 1)

need to  include type and location too

ex: 
 "type": "Microsoft.Sql/managedInstances,
  "location": "<>",

Reply:
Thanks, fixed